students/JasneetChandok/lesson03/assignment/src/basic_operations.py

```diff
@@ -0,0 +1,79 @@
+from peewee import *
+
+database = SqliteDatabase("customers.db")
+database.connect()
+database.execute_sql("PRAGMA foreign_keys=ON;")
+
+
+class BaseModel(Model):
+    class Meta:
+        database = database
+
+
+class Customer(BaseModel):
+    customer_id = IntegerField(primary_key=True)
+    name = CharField(max_length=50)
+    lastname = CharField(max_length=50)
+    home_address = CharField(max_length=50)
+    phone_number = CharField(max_length=50)
+    email = CharField(max_length=50)
+    status = CharField(max_length=10)
+    credit_limit = IntegerField()
+
+
+def add_customer(customer_id, name, lastname, home_address, phone_number, email, status, credit_limit):
+    try:
+        with database.transaction():
+            customer = Customer.create(
+                customer_id=customer_id,
+                name=name,
+                lastname=lastname,
+                home_address=home_address,
+                phone_number=phone_number,
+                email=email,
+                status=status,
+                credit_limit=credit_limit,
+            )
+            customer.save()
+    except Exception as e:

```

Comment:
Similar to before, you don't want to capture the general Exception. In this case, I believe you will want to look for peewee.DoesNotExist